R/tm_variable_browser.R

```diff
@@ -1154,98 +1155,123 @@ render_tab_header <- function(dataset_name, output, data) {
 #' @param parent_dataname (`character`) the name of a parent `dataname` to filter out variables from
 #' @inheritParams render_tabset_panel_content
 #' @keywords internal
-render_tab_table <- function(dataset_name, parent_dataname, output, data, input, columns_names) {
+render_tab_table <- function(dataset_name, parent_dataname, output, data, input, columns_names, plot_var) {
```

Comment:
Please add documentation of a new argument

Reply:
This variable's documentation is already defined and inherited from `render_tabset_panel_content` and is already updated on `man/render_tab_table.Rd`

Reply:
Thats fine. Sorry, didnt realize this